### DISCUSSION TITLE
Auto database backup function

### DISCUSSION THREAD

**Discussion by gth7754125** [Difegue/LANraragi#770] [Ideas]
Is the Redis backup mode changed to aof ? i saw a aof file and a folder named `appendonlydir` on the database's folder, and my onedrive warn me about it because i put the whole database folder to cloud drive and it has been frequently modified. 

So can we add a feature to generate a `backup.json` file regularly so that i can just backup this file instead ?  

**App Version**
LRR 0.8.81 with docker container

**Comment by Difegue** [comment]
The server currently uses both AOF and RDB, with AOF serving as an instasave with regular RDB backups.  
You should just be able to backup the rdb file and ignore the appendonly one.  

If you want regular `backup.json` files, consider running `perl ./script/backup` in the container at regular intervals.

**Reply by gth7754125** [reply]
Thanks for helping. 
Eh... Actually, i need this function because sometimes some archives may change or move to other directories or rename etc... 
Then LRR may lose their metadata when using `clean up the database` and i have to re-add them painfully if the archive don't have `info.json` inside.
 ORZ, or some better way to solve it?
